vue_app/dist/global.css

```diff
@@ -0,0 +1,81 @@
+@charset "UTF-8";

```

Comment:
recommend shifting file to own module, if is downloaded from elsewhere would require directly ffrom node modules so that updates are delightful

Reply:
recommend gitignoring all dist code => leave it up to the machine to build it later